src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -1289,4 +1290,1373 @@ void EIRPlantLoopHeatPump::oneTimeInit(EnergyPlusData &state)
         this->oneTimeInitFlag = false;
     }
 }
+
+// From here on, the Fuel Fired Heat Pump module EIRFuelFiredHeatPump
+// Enum string definitions
+static constexpr std::array<std::string_view, static_cast<int>(EIRFuelFiredHeatPump::OATempCurveVar::Num)> OATempCurveVarNamesUC = {"DRYBULB",
+                                                                                                                                    "WETBULB"};
+static constexpr std::array<std::string_view, static_cast<int>(EIRFuelFiredHeatPump::WaterTempCurveVar::Num)> WaterTempCurveVarNamesUC = {
+    "ENTERINGCONDENSER", "LEAVINGCONDENSER", "ENTERINGEVAPORATOR", "LEAVINGEVAPORATOR"};
+static constexpr std::array<std::string_view, static_cast<int>(EIRFuelFiredHeatPump::DefrostType::Num)> DefrostTypeNamesUC = {"TIMED", "ONDEMAND"};
+
+void EIRFuelFiredHeatPump::doPhysics(EnergyPlusData &state, Real64 currentLoad)
```

Comment:
🤣 I demand that 80% of the functions in EnergyPlus be renamed `doPhysics` and differentiated by overloading.

Reply:
Leaving this as it is for now since this function was inherited (and overridden) from the parent plant loop one that still uses `doPhysics()`. Probably worth a change together (maybe later?)

Reply:
Yes, leave it for now, I think mostly a tongue-in-cheek conversation.

Reply:
Thanks for outing me.